### ISSUE TITLE
WebDAV: stream_get_contents() expects parameter 2 to be long, string given

### ISSUE THREAD

**Issue by H4ndl3** [nextcloud/server#680]
### Steps to reproduce
1. Install Nextcloud 9.0.53
2.  Use WebDAV to sync files from Synology Diskstation to Nextcloud (URL: cloud.example.com/remote.php/dav/files/Username)
### Expected behaviour

No error shows up.
### Actual behaviour

Files get uploaded but log gets filled with the following error:

> stream_get_contents() expects parameter 2 to be long, string given at \/var\/www\/cloud\/3rdparty\/sabre\/http\/lib\/Message.php#81
### Server configuration

**Operating system**: Debian 8.5

**Web server:** nginx/1.6.2 

**Database:** MySQL 10.0.25-MariaDB

**PHP version:** PHP 5.6.24

**Nextcloud version:** 9.0.53

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated from ownCloud 9.0.2

**Where did you install Nextcloud from:** Tarball from official website

The same error seems to be present in ownCloud, see here: https://central.owncloud.org/t/after-migration-to-a-new-server/1185/6

**Comment by icewind1991** [comment]
Can you provide a stacktrace for the error?

**Comment by H4ndl3** [comment]
How can I obtain one?

**Comment by calis2002** [comment]
I got the same error when i upload a file from my ubuntu server via dav2fs to my nextcloud server.
Unfortunatelly there is no stacktrace available. Its just that one line error message.
`{"reqId":"lRhCa5umZnaaLxW\/gE4k","remoteAddr":"x.x.x.x","app":"PHP","message":"stream_get_contents() expects parameter 2 to be integer, string given at \/var\/www\/nextcloud\/3rdparty\/sabre\/http\/lib\/Message.php#81","level":3,"time":"2016-08-19T01:01:25+02:00","method":"MKCOL","url":"\/remote.php\/webdav\/test\/tmp\/","user":"xxx"}`

**Comment by H4ndl3** [comment]
Still present in Nextcloud 10.0.

**Comment by pamart** [comment]
I got this error too in the same situation, using CloudSync on my Synology. Files seem to be synced correctly, but my log filles growing more than 150 MB in only one day.

Server configuration
Operating system: Debian 8.5
Web server: nginx/1.10.1
Database: MySQL 10.0.27-MariaDB
PHP version: PHP 7.0.10-1~dotdeb+8.1
Nextcloud version: 10.0.0

**Comment by koreywithak** [comment]
I get a similar error all the time. My logs are filled with them:
`Error  PHP  stream_get_contents() expects parameter 2 to be integer, string given at /var/www/nextcloud/3rdparty/sabre/http/lib/Message.php#81`
I got this last one when accessing files either via the Android app and/or the web interface.

**Comment by giacomo892** [comment]
Still present in 10.0.2

**Comment by tx7** [comment]
Error persist in NC 11.0

**Comment by aronovgj** [comment]
I have this using FolderSync on android.
NC11
nginx 1.11.5
php 7.0.8
mariadb 10.0.28

`expects parameter 2 to be integer, string given`

Best I can do:
me uploading a lot of backups from android

nginx/access.log
```
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:20 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213810.tar.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:21 +0100] "PUT /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:21 +0100] "GET /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:21 +0100] "GET /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:21 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:21 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:22 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:26 +0100] "PUT /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:26 +0100] "GET /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:26 +0100] "GET /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:27 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:27 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:27 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-b607967332629e49d8d29826c6701418.apk.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:27 +0100] "PUT /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:27 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:28 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:28 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:28 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:28 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:31 +0100] "PUT /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:31 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:32 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:32 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:32 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:32 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.properties HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:32 +0100] "PUT /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:32 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:33 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:33 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:33 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:33 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:36 +0100] "PUT /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:36 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:37 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:37 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:37 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:37 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213658.tar.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:39 +0100] "PUT /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:39 +0100] "GET /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:39 +0100] "GET /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:39 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:39 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:40 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:44 +0100] "PUT /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:44 +0100] "GET /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:45 +0100] "GET /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:45 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:45 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:45 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-f96cdde1192589ec6b5d1218eaa00cfd.apk.gz HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:45 +0100] "PUT /remote.php/webdav/android/titanium/someapp-20161214-213940.properties HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:46 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213940.properties HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:46 +0100] "GET /remote.php/webdav/android/titanium/someapp-20161214-213940.properties HTTP/1.1" 404 11612 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:46 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213940.properties HTTP/1.1" 301 178 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - - [15/Dec/2016:01:50:46 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213940.properties HTTP/1.1" 401 317 "-" "OkHttp" "-"
xxx.xxx.xxx.xxx - me [15/Dec/2016:01:50:46 +0100] "PROPFIND /remote.php/webdav/android/titanium/someapp-20161214-213940.properties HTTP/1.1" 404 11612 "-" "OkHttp" "-"
```
No nginx errors.

NC with log level 0:
```
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-20161214-213700.tar.gz could not be located	2016-12-15T01:54:28+0100
Error	PHP	stream_get_contents() expects parameter 2 to be integer, string given at /var/www/nextcloud/3rdparty/sabre/http/lib/Message.php#81	2016-12-15T01:54:28+0100
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2016-12-15T01:54:28+0100
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-20161214-213700.tar.gz could not be located	2016-12-15T01:54:27+0100
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2016-12-15T01:54:27+0100
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-20161214-213700.tar.gz could not be located	2016-12-15T01:54:24+0100
Error	PHP	stream_get_contents() expects parameter 2 to be integer, string given at /var/www/nextcloud/3rdparty/sabre/http/lib/Message.php#81	2016-12-15T01:54:24+0100
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2016-12-15T01:54:23+0100
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-20161214-213700.tar.gz could not be located	2016-12-15T01:54:23+0100
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2016-12-15T01:54:23+0100
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-20161214-213700.properties could not be located	2016-12-15T01:54:23+0100
Error	PHP	stream_get_contents() expects parameter 2 to be integer, string given at /var/www/nextcloud/3rdparty/sabre/http/lib/Message.php#81	2016-12-15T01:54:23+0100
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2016-12-15T01:54:22+0100
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-20161214-213700.properties could not be located	2016-12-15T01:54:22+0100
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2016-12-15T01:54:22+0100
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-20161214-213700.properties could not be located	2016-12-15T01:54:18+0100
Error	PHP	stream_get_contents() expects parameter 2 to be integer, string given at /var/www/nextcloud/3rdparty/sabre/http/lib/Message.php#81	2016-12-15T01:54:18+0100
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2016-12-15T01:54:18+0100
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-20161214-213700.properties could not be located	2016-12-15T01:54:18+0100
Debug	webdav	Sabre\DAV\Exception\NotAuthenticated: HTTP/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured	2016-12-15T01:54:17+0100
Debug	webdav	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name android/titanium/someapp-8636dfe98278dbf95b32320f8b2e1944.apk.gz could not be located
```

**Comment by aronovgj** [comment]
Does not seem to be an android issue. Removed a folder in the web UI and got this.

**Comment by cowai** [comment]
I have the same issue. But using no sync clients. Its only happening in external_storage (smb).

**Comment by Xuanwo** [comment]
The same on NC 11

**Comment by benhartwich** [comment]
I´ve the same problem at the web client and sync clients of nextcloud in the latest version 11.0.2. 
`stream_get_contents() expects parameter 2 to be integer, string given at /var/kunden/data/nextcloud/3rdparty/sabre/http/lib/Message.php#81`

**Comment by msrv** [comment]
Same issue here, using 11.0.2 and latest nginx on Debian Jessie. Error occurs while using FolderSync (Android) or the Desktop Client on Windows 10.

`stream_get_contents() expects parameter 2 to be long, string given at /srv/www/nextcloud/3rdparty/sabre/http/lib/Message.php#81`

**Comment by Kinamo** [comment]
Not Android related, more of a path problem or something? I have a customer doing syncs and we get this error all over the screen, however the folders are created.
Version 11.0.2.7!

`{"reqId":"XXX","remoteAddr":"XXX","app":"PHP","message":"stream_get_contents() expects parameter 2 to be long, string given at \/var\/www\/XXX\/nextcloud\/3rdparty\/sabre\/http\/lib\/Message.php#81","level":3,"time":"2017-03-30T13:33:27+00:00","method":"MKCOL","url":"\/remote.php\/webdav\/FOLDERNAMES","user":"XXX","version":"11.0.2.7"}`

**Comment by pamart** [comment]
I use my Synology with Cloud Sync and i have this error too since very long time now...

**Comment by Bo0oM** [comment]
I have the same issue.

**Comment by Dennis14e** [comment]
Any update on this? I have the same issue.

**Operating system:** Ubuntu Server 17.04
**Web server:** nginx/1.10.3
**Database:** MySQL 10.1.22-MariaDB
**PHP version:** PHP 7.1.4
**Nextcloud version:** 11.0.3.2

**Synology DSM version:** 6.1.1-15101
**Synology Cloud Sync version:** 2.3.1-0946

`{"reqId":"XXX","remoteAddr":"XXX","app":"PHP","message":"stream_get_contents() expects parameter 2 to be integer, string given at \/var\/www\/dennis14e.de\/cloud\/3rdparty\/sabre\/http\/lib\/Message.php#81","level":3,"time":"2017-05-01T19:26:54+00:00","method":"PROPFIND","url":"\/remote.php\/dav\/files\/Dennis14e\/FOLDERPATH","user":"Dennis14e","version":"11.0.3.2"}
{"reqId":"XXX","remoteAddr":"XXX","app":"PHP","message":"stream_get_contents() expects parameter 2 to be integer, string given at \/var\/www\/dennis14e.de\/cloud\/3rdparty\/sabre\/http\/lib\/Message.php#81","level":3,"time":"2017-05-01T19:26:55+00:00","method":"PROPFIND","url":"\/remote.php\/dav\/files\/Dennis14e\/FOLDERPATH","user":"Dennis14e","version":"11.0.3.2"}`

**Comment by MorrisJobke** [comment]
> Edit: Seems like nextcloud/3rdparty@e724ba5#diff-37d40e6ed6a322af9fa2c40310e7229fL78 (Line 78) fixed it. So would the problem be resolved soon?

Fixed by #3648 with upcoming Nextcloud 12
